perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -293,9 +326,145 @@ def _MakeSwiftCommandPrefix(auth_url, tenant_name, username, password):
   return ' '.join(options)
 
 
+def _ParseMultiStreamResults(raw_result, metadata=None):
+  """Read results from the api_multistream worker process.
+
+  Args:
+    raw_result: string. The stdout of the worker process.
+    metadata: dict. Base sample metadata
+
+  Returns:
+    A list of sample.Sample objects.
+  """
+
+  if metadata is None:
+    metadata = {}
+
+  records = json.loads(raw_result)
+  results = []
+
+  for record in records:
+    metric = 'Multi-stream %s KB %s' % (record['size'], record['operation'])
+    md = metadata.copy()
+    md['stream_num'] = record['stream_num']
+    md['object_size_b'] = record['size']
+
+    results.append(sample.Sample(metric,
+                                 record['latency'],
+                                 'sec',
+                                 md,
+                                 record['start_time']))
+
+  return results
+
+
+def _StringToBytes(string):
```

Comment:
move this function and the next one to a utility file, to be shared and to benefit all other benchmarks. This is general enough to be out of this file.